util/find-doc-nits

```diff
@@ -1073,15 +1092,16 @@ sub checkflags {
     close CFH;
 
     # See what's in the command not the manpage.
-    my @undocced = sort grep { !defined $docopts{$_} } keys %cmdopts;
+    my @undocced = sort grep { !defined $docopts{$_} } @cmdopts;
     foreach ( @undocced ) {
-        next if /-/; # Skip the -- end-of-flags marker
+        next if $cmd eq "openssl" && $_ eq "help";
         err("$doc: undocumented option -$_");
     }
 
     # See what's in the command not the manpage.
-    my @unimpl = sort grep { !defined $cmdopts{$_} } keys %docopts;
+    my @unimpl = sort grep { my $e = $_; !(grep /^$e$/, @cmdopts) } keys %docopts;
```

Comment:
`/^$e$/` should be `/^\Q$e\E$/`, so `$e` doesn't get interpreted as a regex

Reply:
Thanks for this hint, done.